designer/server/src/main/scala/pl/touk/nussknacker/ui/process/deployment/DeploymentServiceImpl.scala

```diff
@@ -269,28 +267,24 @@ class DeploymentServiceImpl(dispatcher: DeploymentManagerDispatcher,
     }
   }
 
-  private def getStateFromEngine(deploymentManager: DeploymentManager, processIdWithName: ProcessIdWithName)
-                                (implicit ec: ExecutionContext, freshnessPolicy: DataFreshnessPolicy): Future[WithDataFreshnessStatus[Option[ProcessState]]] = {
-    lazy val failedToGetResult =
-      WithDataFreshnessStatus(
-        Option(SimpleProcessStateDefinitionManager.errorFailedToGet),
-        cached = false)
+  private def getStateWithResolvedInconsistency(deploymentManager: DeploymentManager, processIdWithName: ProcessIdWithName, lastAction: Option[ProcessAction])
```

Comment:
I would not mention here about inconsistency. We moved inconsistency into lower levels so now it is technical detail. We can think about sth like "getStateFromEngineConsideringLastAction" or sth in this direction. WDYT?